### ISSUE TITLE
Window not showing

### ISSUE THREAD

**Issue by tinloaf** [atom/atom#5920]
I'm using atom 0.186.0-5f7015f on a Gentoo Linux. I currently have the problem that although atom is running as a process (and even xwininfo is showing the windows), no window is displayed. I am starting atom as 

```
atom -l tmp/atom.log -f -w --safe
```

The call immediately returns, so a window should be created. I suspect that even if my window manager was doing something strange (I'm using awesome), some sort of logfile should be created in tmp/atom.log, but that file is not created. Also, plugins should not get in the way because of the --safe switch. I even completely moved my ~/.atom - folder to another location to get a clean start.

Thus, I'm assuming atom is doing something wrong here. Since there is no output whatsoever, no log is created, nothing shows up in my syslog or Xorg's log, I'm at a loss to do further debugging. Any idea what's wrong here?

**Comment by Zireael07** [comment]
Might be same as #5900 

Look for a teeny-tiny window...

**Comment by tinloaf** [comment]
No, it's not that bug: If it was a 1x1 window, it would still be showing in my task bar. Aside from that, this is not my first launch of Atom, and also I'm using a tiling window manager, which should cause Atom to go fullscreen automatically. :( Any other ideas?

**Comment by tinloaf** [comment]
Some more info: Under /tmp, a file 'atom.sock' is created, as well as a directory 'Atom Crashes', which is empty.

**Comment by lkarthee** [comment]
It is not consistently reproducible - but happens randomly. I have first noticed in 0.186.0 build and seen in 0.187.0 build. I am using Ubuntu 14.04 LTS. 

I did some trial and error - when this happens: 
 - if i try to open another window from commandline like another folder using `atom` the new window opens normally.
 -  If try to quit it will close normally. 
 -  If i reopen the folder whose window is like 1px wide it will with same dimension 1px wide. 
Uploading a screenshot.

![screenshot from 2015-03-11 17 09 30](https://cloud.githubusercontent.com/assets/2712395/6595702/a17d2db6-c812-11e4-9e39-ea46fdbf7013.png)
